src/components/button-minor/button-minor.spec.tsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
+import Icon from "components/icon";
```

Comment:
Incorrect import path?

Reply:
Corrected to `import Icon from "../icon";`. Other components using the same incorrect path will be corrected in another ticket.